.github/workflows/add_label_automerge.yml

```diff
@@ -1,21 +1,23 @@
 name: Add label on auto-merge enabled
+
```

Comment:
I don't think we have automerge set up...

It would be nice to have- @ckadner let me know if you have experience with turning it on and we can hit a huddle and get this going

Reply:
Just set up 👍